docs/UserGuide.md

```diff
@@ -189,4 +189,11 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
+**Group** | `group GROUP_NAME` <br> e.g., `group CS2103 Project Group`
+**Groups** | `groups`
+**Delete Group** | `deleteG g/GROUP_INDEX`<br> e.g., `deleteG g/2`
+**Adding people to a group** | `addG PERSON_INDEX_1, PERSON_INDEX_2.. g/GROUP_INDE`<br> e.g.,`addG 1, 2, 3 g/2`
+**Join** | `join PERSON_INDEX g/GROUP_INDEX_1[,GROUP_INDEX_2]...`<br> e.g., `join 2 g/1,3,6,8`
+**Mates** | `mates GROUP_INDEX`<br> e.g., `mates 7`
+**Destroy** | `destroy GROUP_INDEX`<br> e.g., `destroy 2`
```

Comment:
I think this is intended to be the command that removes a person from all groups. Maybe it should be `PERSON_INDEX` instead.

Reply:
![Screenshot 2021-09-26 at 11 13 42 PM](https://user-images.githubusercontent.com/77202230/134813663-b38ba85c-dd3f-4aa6-a3a6-d1aa5e6622b4.png)

Reply:
I think there is an error in the Share Google Docs @asherhy